troposphere/static/js/components/modals/instance/launch/components/Image.react.js

```diff
@@ -13,13 +13,31 @@ define(
     return React.createClass({
       displayName: "Image",
 
+      getInitialState: function(){
+          let image = this.props.image;
+          let versionList = null;
+          let active = false;
+            if (image) {
+                versionList = image.get('versions');
+
+                if (versionList.length > 0) {
+                    active = true;
+                }
```

Comment:
I believe you could just set `active` to the result of `versionList.length > 0` and avoid the need for the if altogether.